packages/query-devtools/src/Devtools.tsx

```diff
@@ -3397,5 +3436,5 @@ const stylesFactory = (theme: 'light' | 'dark') => {
   }
 }
 
-const lightStyles = stylesFactory('light')
-const darkStyles = stylesFactory('dark')
+const lightStyles = (css) => stylesFactory('light', css)
+const darkStyles = (css) => stylesFactory('dark', css)
```

Comment:
```suggestion
const lightStyles = (css: typeof goober['css']) => stylesFactory('light', css)
const darkStyles = (css: (typeof goober)['css']) => stylesFactory('dark', css)
```

Types are erroring could you fix the types like so